app/scripts/controllers/rif/rns/resolver/index.js

```diff
@@ -215,12 +150,93 @@ export default class RnsResolver extends RnsJsDelegate {
     });
   }
 
+  async suscribeToMulticriptoEvents(domainName) {
+    const node = namehash.hash(domainName);
+    console.debug('===============================================================================')
+    const topicRsk = {
```

Comment:
lets put a more representative variable name, like `rnsTopic` or `rnsMulticryptoTopic`. `topicRsk `is too wide and does not gives information about the contract events are related with rns resolver

Reply:
This logic has change, now we dont use topics anymore